src/pages/terms-and-conditions.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Terms and conditions"
+date: "2020-11-24"
+---
+
+# Terms and conditions
+
+Paying customers of Snowplow Analytics Ltd. and Snowplow Analytics Inc. may download, make copies of and distribute the technical documentation that we publish on this wiki internally within their business as reasonably necessary in connection with their agreed use of our paid-for products.
```

Comment:
Let me review this with legal, I think we can simplify this.

Reply:
Please replace this entire text with just:

---

The documentation on this site is published under the terms of the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International Public Licence available at:

[https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.)